api/routes/web.php

```diff
@@ -14,3 +14,6 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+
+Route::resource('resource', 'resourceController');
```

Comment:
It looks like this might be better as `Route::apiResource`, but that doesn't matter much.

https://laravel.com/docs/5.5/controllers#restful-partial-resource-routes

Because we don't need the `GET` requests to `/api/resource/create` etc.

Reply:
Good catch, I didn't know that was a thing. I'll switch it over later today.